courses/views_test.py

```diff
@@ -50,6 +58,44 @@
 EXAMPLE_URL = "http://example.com"
 
 
+@pytest.fixture(autouse=True)
+def _raise_nplusone(request):
+    if request.node.get_closest_marker("skip_nplusone"):
+        yield
+    else:
+        with Profiler():
+            yield
```

Comment:
So we can reuse this in other test files, define this at the root of the project, but without `autouse-True` and instead in this module and any others that need it use:
```python
pytestmark = pytest.mark.usefixture("raise_nplusone")
```

Reply:
Yes yes.
Should have to start.

Reply:
https://github.com/mitodl/mitxonline/pull/1962/files#diff-4ff824f0186901e292c8550b4150c3c05f94653f487c498a3dc384a4ff998da5 moved it.